server/controllers/review.js

```diff
@@ -0,0 +1,63 @@
+const ObjectId = require("mongoose").Types.ObjectId;
+const asyncHandler = require("express-async-handler");
+const Profile = require("../models/Profile");
+const User = require("../models/User");
+const Review = require("../models/Review");
+
+// @route POST /review/:id
+// @Given parameters passed in, create a review.
+exports.createReview = asyncHandler(async (req, res, next) => {
+  const userId = req.body.id;
```

Comment:
You should be able to get the current user id with req.user.id when protecting a route

Reply:
changed to req.user.id